ecosystem/sep-0009.md

```diff
@@ -95,3 +103,53 @@ Where possible we use field names from [schema.org](https://schema.org/Person).
 ### Addresses
 
 Address formatting varies widely from country to country and even within each country. See [here](https://stackoverflow.com/questions/11160192/how-to-parse-freeform-street-postal-address-out-of-text-and-into-components) for details. Rather than attempting to create a field for each possible part of an address in every country, this protocol takes a middle of the road approach. Address fields that are fairly universal can be encoded with the `country_code`, `state_or_province`, `city`, and `postal_code` fields. Full addresses, however, should be encoded as a single multi-line string in the `address` field. This allows any address in the world to be represented with a limited number of fields. If address parsing is necessary, parsing will be easier since the country, city, and postal code are already separate fields.
+
+### Beneficial Owners
+* Individual Beneficial Owners - individual (person)  which holds ownership.
+* Entity Beneficial Owner - entity (organization)  which holds ownership. 
+* Control Person - an important person in a corporation. Control persons include senior managers, members of the board of directors, and officers such as the CEO and CFO. Control persons are able to use both their authority and their influence to make decisions on the corporation's activities. A control person is also called an affiliated person.
+
+### Control Person Title List 
```

Comment:
Would a request containing a `title` value not given here be rejected? If not, we should remove this list.